services/src/main/webapp/WEB-INF/web.xml

```diff
@@ -58,6 +58,10 @@
             <web-resource-name>Public</web-resource-name>
             <url-pattern>/ArquillianServletRunner/*</url-pattern>
         </web-resource-collection>
+        <web-resource-collection>
```

Comment:
Ack -- this basically disables authentication completely for web sockets. I get why this might be necessary, but I think this will need to be fixed.

I think the client will need to pass an authentication token and then validate it with keycloak. I can look at this some next week if we need to investigate how to do this.